CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set(_ct_min_cmake_version "3.14")
+set(_ct_min_cmake_version "3.14" CACHE STRING "" FORCE)
```

Comment:
I think setting the type to `INTERNAL` would be better here

Reply:
I agree. I was not aware of the `INTERNAL` type, so I appreciate this!